runtime/darwinia/src/pallets/relayer_game.rs

```diff
@@ -4,8 +4,8 @@ pub use darwinia_relayer_game::Instance1 as EthereumRelayerGameInstance;
 use frame_support::traits::LockIdentifier;
 // --- darwinia-network ---
 use crate::*;
-use darwinia_relay_primitives::AdjustableRelayerGame;
```

Comment:
Another place related to this companion is:

```rs
Call::EthereumBacking(..) | Call::EthereumRelay(..) | Call::EthereumRelayAuthorities(..)
```

Or

```rs
Call::EthereumBacking(..) | Call::EthereumRelay(..) | Call::EcdsaRelayAuthority(..)
```

Please check it again.